sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -8,5 +25,64 @@ package life.qbic.portal.sampletracking.components.projectoverview
  * @since 1.0.0
  *
 */
-class ProjectOverviewView {
+class ProjectOverviewView extends VerticalLayout{
+
+    private Label titleLabel
+    private ProjectOverviewViewModel viewModel
+
+    private Grid<Project> projectGrid
+    private Button detailsButton
+    private Button downloadButton
+
+    ProjectOverviewView(ProjectOverviewViewModel viewModel){
+        this.viewModel = viewModel
+
+        initLayout()
+        fillGrid()
+    }
+
+    private void initLayout(){
+        titleLabel = new Label("Project Overview")
+        titleLabel.addStyleName(ValoTheme.LABEL_LARGE)
+        projectGrid = new Grid<>()
+
+        detailsButton = new Button("Show Details",VaadinIcons.INFO_CIRCLE_O)
+        detailsButton.addStyleName(ValoTheme.LABEL_LARGE)
+        detailsButton.enabled = false
+        downloadButton = new Button("Download Batch Data",VaadinIcons.DOWNLOAD)
+        downloadButton.addStyleName(ValoTheme.LABEL_LARGE)
+        downloadButton.enabled = false
+
+        HorizontalLayout buttonLayout = new HorizontalLayout(detailsButton,downloadButton)
+
+        this.addComponents(titleLabel,buttonLayout,projectGrid)
+    }
+
+    private void fillGrid(){
+        projectGrid.addColumn({ it.projectCode })
+                .setCaption("Project Code").setId("ProjectCode")
+        Grid.Column<Project,String> descriptionColumn = projectGrid.addColumn({ it.projectDescription })
+                .setCaption("Description").setId("Description")
+        //todo add progressbar here
+        Grid.Column<Project, FailedSamplesRatio> failedSamplesColumn = projectGrid.addColumn({ it.failedSamples })
+                .setCaption("Failed Samples").setId("FailedSamples")
+        //failedSamplesColumn.setRenderer(failedSamples -> failedSamples, new TextRenderer())
+        Grid.Column<Project,Date> dateColumn = projectGrid.addColumn({ it.lastUpdate })
```

Comment:
Nit: The variable name could be more explicit to avoid confusion if a seperate date column(e.g. startDate) or something is required down the line
```suggestion
        Grid.Column<Project,Date> updateColumn = projectGrid.addColumn({ it.lastUpdate })
```